cityofaustin/atd-vz-data
# Write SQL to backfill the crash_pk column for crash recommendations and notes

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18504

## Testing
**URL to test:** 
Local data model

**Steps to test:**
1. Start up your local data model. To truly test this you need to import a decent chunk of crashes using the cris_import script. I used the 10 year extracts John linked in this [slack thread](https://austininnovation.slack.com/archives/CM9MK950S/p1723046093254839?thread_ts=1722874844.778289&cid=CM9MK950S) 
2. After importing all the crashes, run the SQL commands in this PR to backfill the `crash_pk` column in the crash notes and recommendations tables. If you get an error saying there are still null values in the `crash_pk` column of recommendations/crash_notes then thats because your import didnt contain all of the crashes with recommendations or notes so, you can kinda basically ignore that maybe remove those commands with the faith that John and I both tested with full crash imports and we will make sure this is right for launch :crossed_fingers: 
3. In your VZE test going to different crash records using their cris crash ids and seeing the corresponding notes and recommendations there on the page.


---
#### Ship list
